<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maincraft Skin to PDF</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>398</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>470</width>
        <height>398</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>359</height>
        </size>
       </property>
       <attribute name="title">
        <string>Maincraft skin to PDF</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QSpinBox" name="sbSkinFacePadding">
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Skin part padding (mm)</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Margins (mm)</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Page Orientation</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Left/Right</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbSkinMarginHor">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Top/Bottom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbSkinMarginVert">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>PNG Skin file</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Mob name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="leMobName">
              <property name="maxLength">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Paper Size</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Skin face padding (mm)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
               <widget class="QSpinBox" name="sbGridSize">
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Grid Size(mm)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="leSkinPNGFile">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSkinBrowsePNG">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QComboBox" name="cbSkinPaperSize"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="4" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QSpinBox" name="sbSkinPartPadding">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="6" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QComboBox" name="cbSkinPageOrientation">
                <item>
                 <property name="text">
                  <string>Portrait</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Landscape</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbSkinSavePDF">
              <property name="text">
               <string>Save to PDF</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>359</height>
        </size>
       </property>
       <attribute name="title">
        <string>Grid Builder</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Grids Size(mm)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLineEdit" name="leGridGridsSizes">
                <property name="text">
                 <string>5,3,7,10</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_13">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Grids padding (mm)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Page Orientation</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QComboBox" name="cbGridPageOrientation">
                <item>
                 <property name="text">
                  <string>Portrait</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Landscape</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_16">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QSpinBox" name="sbGridPadding">
                <property name="minimum">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_15">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Paper Size</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QComboBox" name="cbGridPaperSize"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_14">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Margins (mm)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Left/Right</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbGridMarginHor">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_11">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Top/Bottom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbGridMarginVert">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_12">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item>
             <spacer name="horizontalSpacer_17">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Save to PDF</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
